### DISCUSSION TITLE
Managing Azure assets

### DISCUSSION THREAD

**Discussion by the-d-b** [netbox-community/netbox#17531] [General]
Hi all.  I'm relatively new to netbox.  I started my journey looking for an ipam.  Instead of just simply an ipam I found netbox.  Honestly I am more interested in netbox now for tons of other regions besides ipam.  Device management, racks, areas, locations, connections.  All really awesome.  And I have several favorite plugins that I am leveraging to extend functionality.  This is just an awesome tool.

My question is though - we run a hybrid cloud environment.  Many of these features are perfect for our on-premise data center.  But when it comes to Azure assets I have not yet looked at how this tool would help.  I'm interested in things like NSGs, App Gateways, load balancers, traffic managers, Azure service busses, vnets, expressroutes, etc.

Is anybody using netbox to document public cloud assets and is it working well?  Are there any public cloud specific plugins or are they even necessary.  When dealing with public cloud how do you document connections?

I'm using the topology-views plugin and find it to be amazing and valuable.  Any tips on how to configure Azure connections to be able to use it?  Is there a way to do a 'sync' to capture Azure assets automatically?

**Comment by mbrksntrk** [comment]
I'm also looking for the same.

**Comment by rduffner** [comment]
There's https://github.com/sander110419/Netbox-Azure-Networks

That will help a bit at least.
